.eslintrc.json

```diff
@@ -44,7 +44,17 @@
           }
         ],
         "@typescript-eslint/semi": 0,
-        "import/no-unresolved": "error"
+        "import/no-unresolved": "error",
+        "prettier/prettier": [
+          "error",
+          {
+            "trailingComma": "es5",
+            "singleQuote": true,
+            "printWidth": 120,
+            "semi": false,
+            "arrowParens": "avoid"
```

Comment:
Temporarily disabling single paran rule as it leads to 1000s of additional changes and will screw up current development